sydent/sydent.py

```diff
@@ -62,7 +62,7 @@ class Sydent:
         'log.path': '',
         'pidfile.path': 'sydent.pid',
         # db
-        'db.file': 'sydent.db',
+        'db.file': os.environ.get("SYDENT_DB_PATH", "sydent.db"),
```

Comment:
Hmm - I think conventionally environment variables would override the config file if anything, so this is a bit odd (especially as it's for the db path and nothing else).